### ISSUE TITLE
High accuracy 0.843 in train_msc val?

### ISSUE THREAD

**Issue by mokii** [DrSleep/tensorflow-deeplab-resnet#169]
Train for 20K iterations with original parameters in train_msc.py with about 10K VOC training set and val the model.ckpt-8000 for 1449 val images but got the high accuracy:0.843? Is it rational?
step 100
step 200
step 300
step 400
step 500
step 600
step 700
step 800
step 900
step 1000
step 1100
step 1200
step 1300
step 1400
Mean IoU: 0.843

**Comment by DrSleep** [comment]
Did you initialise from the downloaded model?
If so, the released model by DeepLab was pre-trained on trainval, which explains high accuracy on validation.
